src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10635,3 +10677,123 @@ definitions:
         type: object
         additionalProperties:
           type: string
+  RulePaging:
+    type: object
+    required:
+      - list
+    properties:
+      list:
+        type: object
+        required:
+          - pagination
+          - entries
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/RuleEntry'
+  RuleEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Rule'
+  Rule:
+    type: object
+    required:
+      - id
+      - name
+      - conditions
+      - actions
+    properties:
+      id:
+        description: Identifier for the rule
+        type: string
+      name:
+        description: Name of the rule
+        type: string
+      description:
+        description: Description of the rule
+        type: string
+      enabled:
+        description: Whether the rule is enabled
+        type: boolean
+        default: true
+      cascade:
+        description: Whether the rule also applies to subfolders
+        type: boolean
+        default: false
+      asynchronous:
+        description: Whether the rule should be run in the background
+        type: boolean
+        default: false
+      errorScript:
+        description: If the rule should be run in the background then an optional error script can be referenced
+        type: string
+      trigger:
+        description: |
+          The trigger(s) that cause the rule to be activated.
+          * inbound - The rule should be activated when an item enters the folder
+          * update - The rule should be activated when an item within the folder is updated
+          * outbound - The rule should be activated when an item leaves the folder or is deleted
+        type: array
+        items:
+          type: string
+          enum:
+          - inbound
+          - update
+          - outbound
+        default: ["inbound"]
+      conditions:
+        description: The condition that determines whether the actions will be executed for a rule
+        type: '#/definitions/CompositeConditionDefinition'
+      actions:
+        description: The actions for the rule
+        type: array
+        items:
+          $ref: '#/definitions/ActionDefinition'
+  CompositeConditionDefinition:
+    type: object
+    properties:
+      inverted:
+        description: Whether to invert the logic for this condition
```

Comment:
Should we explicitly add a bracketed description e.g. `(if true than "NOT" is applied to the condition)`